en/api.md

```diff
@@ -3701,8 +3701,7 @@ constraints are provided, they are combined using 'AND' semantics.
    -   vid:&lt;vid-integer&gt;:
 
    Match a subnet on a VLAN with the specified
-   VID. Valid values range from 0 through 4094 (inclusive). An untagged
-   VLAN can be specified by using the value "0".
+   VID. Valid values range from 0 through 4094 (inclusive).
```

Comment:
So how is an untagged VLAN specified then?

Reply:
I don't know. I just tried a few alternatives, but I'll speak to the MAAS team. They need to change the API docs anyway.